tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -2403,8 +2388,8 @@ Status StorageManager::set_default_tags() {
   return Status::Ok();
 }
 
-Status StorageManager::group_metadata_consolidate(
-    const char* group_name, const Config* config) {
+Status StorageManagerCanonical::group_metadata_consolidate(
+    const char* group_name, const Config& config) {
```

Comment:
same question regarding lost (unused?) functionalty here as with array_upgrade_version()...

Reply:
IIUC this is addressed by passing in the storage manager's Config one level higher rather than the redundancy here.